native_lib/DDGSyncCrypto.h

```diff
@@ -12,6 +12,8 @@ typedef enum : int {
     DDGSYNCCRYPTO_STRETCHED_PRIMARY_KEY_SIZE = 32,
     DDGSYNCCRYPTO_PROTECTED_SECRET_KEY_SIZE = (crypto_secretbox_MACBYTES + DDGSYNCCRYPTO_STRETCHED_PRIMARY_KEY_SIZE + crypto_secretbox_NONCEBYTES),
     DDGSYNCCRYPTO_ENCRYPTED_EXTRA_BYTES_SIZE = (crypto_secretbox_MACBYTES + crypto_secretbox_NONCEBYTES),
+    DDGSYNCCRYPTO_PUBLIC_KEY = crypto_box_PUBLICKEYBYTES,
```

Comment:
Can you add the _SIZE suffix to these so they match the other constants, please?